packages/redux-dynamic-modules-react/src/DynamicModuleLoader.tsx

```diff
@@ -100,12 +55,14 @@ class DynamicModuleLoaderImpl extends React.Component<
     private _addedModules?: IDynamicallyAddedModule;
     private _providerInitializationNeeded: boolean = false;
     private _store: IModuleStore<any>;
-    private _getLatestState: boolean;
+    private _memoizedRRContext: any;

```

Comment:
Please add some comments why we need to memoizedRRContext, it is ok to call it _memoizedReactReduxContext ... RR is not clear.